recipe/meta.yaml

```
@@ -306,6 +306,7 @@ outputs:
         - tflite_convert --help      # [build_platform == target_platform]
         - toco_from_protos --help    # [build_platform == target_platform]
         - toco --help                # [build_platform == target_platform]
+        - python -c 'import tensorflow as tf; assert tf.test.is_built_with_cuda()'  # [cuda_compiler_version != "None"]
```

Do we want to test the opposite as well (given how `TF_NEED_CUDA` moves around in `build.sh`)?
```suggestion
        - python -c 'import tensorflow as tf; assert tf.test.is_built_with_cuda()'      # [cuda_compiler_version != "None"]
        - python -c 'import tensorflow as tf; assert not tf.test.is_built_with_cuda()'  # [cuda_compiler_version == "None"]
```